Source/ACE.Entity/AceObject.cs

```diff
@@ -11,7 +11,7 @@ namespace ACE.Entity
     [DbTable("ace_object")]
     public class AceObject : ICreatureStats, ICloneable
     {
-        public const ushort WEENIE_MAX = 0xFFFF;
+        public const uint WEENIE_MAX = 199999;
```

Comment:
199999 is a somewhat arbitrary number.  Why this number?  What is the actual maximum we can set it to?  Before we chose 0xFFFF because it fit a ushort.  Are we sure weenies don't have to be constrained to a ushort for some (likely client related) reason?